ccd-definition/CaseEventToComplexTypes/ClaimantResponseLRspec-CUI-nonprod.json

```diff
@@ -0,0 +1,3 @@
+[
+
```

Comment:
We can probably delete this file as we don't use it
Kenneth will be adding it in anyway